app/controllers/eligibility_interface/base_controller.rb

```diff
@@ -26,6 +26,12 @@ def current_namespace
       "eligibility"
     end
 
+    http_basic_authenticate_with name: ENV.fetch("TEST_USERNAME", "test"),
+                                 password: ENV.fetch("TEST_PASSWORD", "test"),
+                                 unless: -> {
+                                   FeatureFlag.active?(:service_open)
+                                 }
```

Comment:
I'd suggest putting this block near the top, below the `after_action` line, so it's easy to see immediately that this controller has authentication.